### DISCUSSION TITLE
Github copilot

### DISCUSSION THREAD

**Discussion by Vette1123** [community/community#30040] [Copilot Conversations]
so i'm a qualified Github student user, however it says on my profile that my usage of copilot will expire on august 22, any help with that?

**Comment by Zoltus** [comment]
> so i'm a qualified Github student user, however it says on my profile that my usage of copilot will expire on august 22, any help with that?

Did you have that date on your school info? Anyways if it expires it should just be re-enabled as far as your github profile has student rights. If your student rights end on github also on august 22 just reapply. Trust me you are gonna be fine 👍🏼

**Reply by Vette1123** [reply]
i really hope so, i just verified my student identity and i got the pro badge on my profile, but still says copilot gonna expire on augest 22

**Reply by Zoltus** [reply]
> i really hope so, i just verified my student identity and i got the pro badge on my profile, but still says copilot gonna expire on augest 22

I can promise u prettymuch 99% u gonna have it, worst case is that u have to reapply.
Also could u mark as answered<3 i would get cool badge

**Comment by Zoltus** [comment]
![ApplicationFrameHost_2022-08-19_20-48-31](https://user-images.githubusercontent.com/23402980/185677895-4b8bcdce-ae59-49c2-bf47-1864d8502a24.png)

**Reply by vsppedro** [reply]
I'm facing this problem too. It was working perfectly until yesterday and I haven't changed a thing. However, I received an email today asking me to update my copilot extension to continue using the service. The problem is that I don't know how to do that. I took this picture from "Billing and Plans", I don't see how I can update the copilot::

![Screenshot from 2022-08-19 22-35-34](https://user-images.githubusercontent.com/18314910/185724288-e0c2b14c-1c77-4761-94f2-bd3c8ff09971.png)

It seems that I don't need to do anything and that I should be able to continue using it for free, but the error persists as you can see in the logs:

```
[INFO] [default] [2022-08-20T01:35:20.187Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-08-20T01:35:20.385Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-08-20T01:35:20.589Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 203 ms
[INFO] [fetch] [2022-08-20T01:35:20.608Z] Please upgrade your Copilot extension to continue using this service.
```

I'm lost. Does anyone have any idea what I'm doing wrong?

EDIT: Reinstalling the extension did the trick: https://stackoverflow.com/questions/72311242/please-upgrade-your-copilot-extension-to-continue-using-this-service

**Comment by ichenglin** [comment]
This issue seems to be resolved in #30048.

> **Message from [veverkap](https://github.com/veverkap)**
> I can confirm that you both will **continue having access** after August 22nd. There is a **bug** (that we are fixing) for some users that shows this incorrect information.

> **Message from [veverkap](https://github.com/veverkap)**
> Would you all mind checking your https://github.com/settings/copilot page to see if this has been resolved. I apologize again for the inconvenience.

**Reply by ossanna16** [reply]
Good catch @ichenglin 🚀
